client/src/components/Icons-Components/All-IconsContainer.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './All-IconsContainer.css';
+import SlackIconSvg from './SlackIconSvg';
```

Comment:
Your paths dont point into any folders. They need to point to the file containing the images

Reply:
OK. Thank you.